components/TripComponents/TripsListElement/TripsListElement.test.tsx

```
@@ -0,0 +1,93 @@
+    render(<TripsListElement {...defaultProps} />);
+    const firstTitle = screen.getAllByText(/Wycieczka poranna/i)[0];
+    fireEvent.click(firstTitle);
+
+    // Aside shows the selected trip via TripDetails (title appears there too)
+    const allOccurrences = screen.getAllByText(/Wycieczka poranna/i);
+    expect(allOccurrences.length).toBeGreaterThan(0);
+  });
+
+  it('closes the trip details when the close button in the aside is clicked', () => {
+    const toggleTrip = jest.fn();
+    render(<TripsListElement toggleTrip={toggleTrip} isTripToggled={true} />);
```

ESLint

[error] Value must be omitted for boolean attribute `isTripToggled`

(react/jsx-boolean-value)